packages/renderer/src/lib/kube/details/EventsTable.spec.ts

```diff
@@ -80,4 +80,28 @@ test('expect the events are displayed', async () => {
   expect(within(rows[2]).getByText('SecondReason')).toBeInTheDocument();
   expect(within(rows[2]).getByText('another-controller')).toBeInTheDocument();
   expect(within(rows[2]).getByText('a second message')).toBeInTheDocument();
+
+  // refresh the component
+  events.push({
+    count: 1,
+    type: 'Normal',
+    reason: 'ThirdReason',
+    reportingComponent: 'a-controller',
+    message: 'a third message',
+    firstTimestamp: new Date(),
+    lastTimestamp: new Date(),
+  });
+  vi.advanceTimersByTime(10_000);
+  await rendered.rerender({ events: events });
```

Comment:
why re-render ? wouldn't it better to make two different tests ?

Reply:
The test name is `expect the events are displayed` so I am not sure to understand the need to re-render it

Reply:
I would like to test that the events are refreshed when the entry is modified (I mean that the `$effect` works as expected)

Reply:
(renaming the test to illustrate that the display must be reactive)

Reply:
Thanks